x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -171,15 +177,17 @@ export function createBulkExecutionEnqueuerFunction({
           executionId: actionToExecute.executionId,
           consumer: actionToExecute.consumer,
           relatedSavedObjects: relatedSavedObjectWithRefs,
+          ...(actionToExecute.apiKeyId ? { apiKeyId: actionToExecute.apiKeyId } : {}),
```

Comment:
I noticed this pattern used in a few places:

```
...(actionToExecute.apiKeyId ? { apiKeyId: actionToExecute.apiKeyId } : {}),
```

I _think_ this can be simplified to 

```
apiKeyId: actionToExecute.apiKeyId,
```

I'm not aware of - but wouldn't be suprised to find - something sensitive to the property not in the object vs the property having `undefined` as the value.